apps/weather.yaml

```diff
@@ -1,7 +1,7 @@
 meta:
-  name: "Weather Stats"
-  description: "A minimal example presenting a local HTTP Proxy connecting to a static website deployed on the provider’s end. The network connecting the requestor and the provider is configured implicitly."
-  author: "Golem Factory"
+  name: "Weather Stats App"
+  description: "Application to get Weather, powered by Golem Network"
```

Comment:
Please leave the original description untouched. Also the author - it looks better in the UI of the app (demo purposes).

Reply:
hmm, so maybe we should include a longer description which is still true to what the app does (the previous description is misleading)